.github/workflows/e2e_courses_and_topics.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   tag-courses_and_topics:
-    uses: hpi-schul-cloud/end-to-end-tests/.github/workflows/e2e.yml@main
+    uses: hpi-schul-cloud/end-to-end-tests/.github/workflows/e2e.yml@BC-7461-mongo6
```

Comment:
this was added only so that the e2e tests called from e2e repo pass
it must be removed before merge